be/src/util/threadpool.cpp

```diff
@@ -610,6 +607,47 @@ void ThreadPool::check_not_pool_thread_unlocked() {
     }
 }
 
+Status ThreadPool::set_min_threads(int min_threads) {
+    if (min_threads <= _max_threads) {
+        _min_threads = min_threads;
+        if (min_threads > _num_threads + _num_threads_pending_start) {
+            int addition_threads = min_threads - _num_threads - _num_threads_pending_start;
+            _num_threads_pending_start += addition_threads;
+            for (int i = 0; i < addition_threads; i++) {
+                Status status = create_thread();
+                if (!status.ok()) {
+                    _num_threads_pending_start--;
+                    LOG(WARNING) << "Thread pool failed to create thread: " << status.to_string();
+                    return status;
+                }
+            }
+        }
+        return Status::OK();
+    }
+    return Status::InternalError("set thread pool min_threads failed");
+}
+
+Status ThreadPool::set_max_threads(int max_threads) {
+    if (_min_threads <= max_threads) {
```

Comment:
Maybe Use short circuit return would make you code more elegant

Reply:
> Maybe Use short circuit return would make you code more elegant

OK, Thank you.